css/style.css

```diff
@@ -159,6 +310,25 @@ p.content-description {
     height: 40px;    
     color: rgba(0, 0, 0, 0.3);
 }
+.content-properties tr {
+    display: flex;
+    justify-content: left;
+}
+.content-properties th:nth-child(1) {
```

Comment:
please enumerate common properties for these children through comma

Reply:
Done